pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -214,15 +298,15 @@ type ResourceSummary struct {
 	// Allocatable represents the resources of a cluster that are available for scheduling.
 	// Total amount of allocatable resources on all nodes.
 	// +optional
-	Allocatable corev1.ResourceList `json:"allocatable,omitempty"`
+	Allocatable ResourceList `json:"allocatable,omitempty"`
 	// Allocating represents the resources of a cluster that are pending for scheduling.
 	// Total amount of required resources of all Pods that are waiting for scheduling.
 	// +optional
-	Allocating corev1.ResourceList `json:"allocating,omitempty"`
+	Allocating ResourceList `json:"allocating,omitempty"`
 	// Allocated represents the resources of a cluster that have been scheduled.
 	// Total amount of required resources of all Pods that have been scheduled to nodes.
 	// +optional
-	Allocated corev1.ResourceList `json:"allocated,omitempty"`
+	Allocated ResourceList `json:"allocated,omitempty"`
```

Comment:
Why modify this field?

Will this change break compatibility?

Reply:
This change has been reverted. I didn't consider compatibility issues. So sorry for changing the wrong place.